spec/jekyll_seo_tag_spec.rb

```diff
@@ -247,6 +247,16 @@
           expect(output).to match(expected)
         end
       end
+
+      context 'with page.image' do
+        let(:site) { make_site('twitter' => site_twitter, 'url' => 'http://example.invalid') }
+        let(:page) { make_page('image' => 'foo.png') }
```

Comment:
Should the image be relative to the root or to the page? I suspect you'd rarely want an image to be at `/2016/01/01/foo/foo.png`?